src/polymer/elements/rules.ts

```diff
@@ -14,3 +14,4 @@
 
 import './iron-flex-layout-classes';
 import './iron-flex-layout-import';
+import './paper-toolbar-v1-to-v2';
```

Comment:
This rule should probably go in the polymer-2-hybrid collection and the polymer-2 collection

Reply:
Added.